### PR TITLE
feat: get font awesome mime type

### PR DESCRIPTION
**Pull request by IsaMic**
|Q              |A  |
|---------------|---|
|Bug fix?       |n|
|New feature?   |y|	
|BC breaks?     |n|	
|Deprecations?  |n|	
|Fixed tickets  |n|	

Add function getFontAwesomeFromMimeType tu use on a file picker (ckeditor) + const

### CODE REVIEW

File: src/Helper/Text/Encoder.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ public function encodeUrl(string $text): string
...
         $b = ['A', 'A', 'A', 'A', 'A', 'A', 'AE', 'C', 'E', 'E', 'E', 'E', 'I', 'I', 'I', 'I', 'D', 'N', 'O', 'O', 'O', 'O', 'O', 'O', 'U', 'U', 'U', 'U', 'Y', 's', 'a', 'a', 'a', 'a', 'a', 'a', 'ae', 'c', 'e', 'e', 'e', 'e', 'i', 'i', 'i', 'i', 'n', 'o', 'o', 'o', 'o', 'o', 'o', 'u', 'u', 'u', 'u', 'y', 'y', 'A', 'a', 'A', 'a', 'A', 'a', 'C', 'c', 'C', 'c', 'C', 'c', 'C', 'c', 'D', 'd', 'D', 'd', 'E', 'e', 'E', 'e', 'E', 'e', 'E', 'e', 'E', 'e', 'G', 'g', 'G', 'g', 'G', 'g', 'G', 'g', 'H', 'h', 'H', 'h', 'I', 'i', 'I', 'i', 'I', 'i', 'I', 'i', 'I', 'i', 'IJ', 'ij', 'J', 'j', 'K', 'k', 'L', 'l', 'L', 'l', 'L', 'l', 'L', 'l', 'l', 'l', 'N', 'n', 'N', 'n', 'N', 'n', 'n', 'O', 'o', 'O', 'o', 'O', 'o', 'OE', 'oe', 'R', 'r', 'R', 'r', 'R', 'r', 'S', 's', 'S', 's', 'S', 's', 'S', 's', 'T', 't', 'T', 't', 'T', 't', 'U', 'u', 'U', 'u', 'U', 'u', 'U', 'u', 'U', 'u', 'U', 'u', 'W', 'w', 'Y', 'y', 'Y', 'Z', 'z', 'Z', 'z', 'Z', 'z', 's', 'f', 'O', 'o', 'U', 'u', 'A', 'a', 'I', 'i', 'O', 'o', 'U', 'u', 'U', 'u', 'U', 'u', 'U', 'u', 'U', 'u', 'A', 'a', 'AE', 'ae', 'O', 'o', ' '];
```
Comment (theus77) [inline src/Helper/Text/Encoder.php:60]: Check your ide encoding?

File: src/Helper/EmsFields.php [hunk 2]
```diff
@@ -75,4 +76,6 @@ final class EmsFields
     const LOG_OPERATION_UPDATE = 'UPDATE';
     const LOG_OPERATION_READ = 'READ';
     const LOG_OPERATION_DELETE = 'DELETE';
+
+    const FONTAWESOME_PREFIX = 'fa ';
```
Comment (theus77) [inline src/Helper/EmsFields.php:80]: I would put this in the Core Bundle

File: src/Helper/EmsFields.php [hunk 3]
```diff
@@ -35,6 +35,7 @@ final class EmsFields
     const ASSET_CONFIG_BORDER_COLOR = '_border_color';
     const ASSET_CONFIG_WATERMARK_HASH = '_watermark_hash';
     const ASSET_CONFIG_GET_FILE_PATH = '_get_file_path';
+    const ASSET_EMSLINK_PREFIX = 'ems://asset:';
```
Comment (theus77) [inline src/Helper/EmsFields.php:38]: Not sure with the name nor the location. What with EMSLINK_ASSET_PREFIX and EMSLINK_OBJECT_PREFIX in the EMS\CommonBundle\Common\EMSLink class?

Comment (IsaMic) [reply src/Helper/EmsFields.php]: Done

**Pull request merged**